client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -0,0 +1,356 @@
+
+class MappedDomainType extends React.Component {
+	getVerticalNavigation() {
+		return (
+			<VerticalNav>
+				{ this.emailNavItem() }
+				{ this.dnsRecordsNavItem() }
+				{ this.domainConnectMappingNavItem() }
+				{ this.transferMappedDomainNavItem() }
+			</VerticalNav>
+		);
+	}
+
+	emailNavItem() {
+		const path = emailManagement( this.props.selectedSite.slug, this.props.domain.name );
+
+		return <VerticalNavItem path={ path }>{ this.props.translate( 'Email' ) }</VerticalNavItem>;
+	}
+
+	dnsRecordsNavItem() {
+		const path = domainManagementDns( this.props.selectedSite.slug, this.props.domain.name );
+
+		return (
+			<VerticalNavItem path={ path }>{ this.props.translate( 'DNS Records' ) }</VerticalNavItem>
+		);
+	}
+
+	domainConnectMappingNavItem() {
+		const { supportsDomainConnect, hasWpcomNameservers, pointsToWpcom } = this.props.domain;
+		if ( ! supportsDomainConnect || hasWpcomNameservers || pointsToWpcom ) {
+			return;
+		}
+
+		const path = domainManagementDomainConnectMapping(
+			this.props.selectedSite.slug,
+			this.props.domain.name
+		);
+
+		return (
+			<VerticalNavItem path={ path }>
+				{ this.props.translate( 'Connect Your Domain' ) }
+			</VerticalNavItem>
+		);
+	}
+
+	transferMappedDomainNavItem() {
+		const { domain, selectedSite, translate } = this.props;
+
+		if ( domain.expired || domain.isSubdomain || ! domain.isEligibleForInboundTransfer ) {
+			return null;
+		}
+
+		const path = domainTransferIn( selectedSite.slug, domain.name, true );
+
+		return (
+			<VerticalNavItem path={ path }>
+				{ translate( 'Transfer Domain to WordPress.com' ) }
+			</VerticalNavItem>
+		);
+	}
+
+	resolveStatus() {
+		const { domain, translate, moment } = this.props;
+		const { expiry } = domain;
+
+		if ( isExpiringSoon( domain, 30 ) ) {
+			const expiresMessage = translate( 'Expires in %(days)s', {
+				args: { days: moment.utc( expiry ).fromNow( true ) },
+			} );
+
+			if ( isExpiringSoon( domain, 5 ) ) {
+				return {
+					statusText: expiresMessage,
+					statusClass: 'status-error',
+					icon: 'info',
+				};
+			}
+
+			return {
+				statusText: expiresMessage,
+				statusClass: 'status-warning',
+				icon: 'info',
+			};
+		}
+
+		if ( ! domain.pointsToWpcom ) {
+			return {
+				statusText: translate( 'Action required' ),
+				statusClass: 'status-error',
+				icon: 'info',
+			};
+		}
+
+		return {
+			statusText: translate( 'Active' ),
+			statusClass: 'status-success',
+			icon: 'check_circle',
+		};
+	}
+
+	renderExpiringSoon() {
+		const { domain, translate, moment } = this.props;
+		const { expiry } = domain;
+
+		if ( ! isExpiringSoon( domain, 30 ) ) {
+			return null;
+		}
+
+		if ( domain.bundledPlanSubscriptionId ) {
+			return (
+				<div>
+					<p>
+						{ translate(
+							'Your domain mapping will expire with your plan in {{strong}}%(days)s{{/strong}}. Please renew your plan before it expires or it will stop working.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **4** times:
`translate( 'Your domain will expire in {{strong}}%(days)s{{/strong}}. Please renew it before it expires or it will stop working.' )` *ES Score: 7*